contracts/main/stablecoin-core/BookKeeper.sol

```diff
@@ -316,6 +316,7 @@ contract BookKeeper is IBookKeeper, ICagable, IPausable, BookKeeperMath, Pausabl
         require(either(_debtShare >= 0, _wish(_stablecoinOwner, msg.sender)), "BookKeeper/not-allowed-stablecoin-owner");
 
         require(either(position.debtShare == 0, _positionDebtValue >= _vars.debtFloor), "BookKeeper/debt-floor");
+        require(_positionDebtValue <= _vars.positionDebtCeiling, "BookKeeper/position-debt-ceiling-exceeded");
```

Comment:
so I guess before openingPosition for 1.15 mil FXD, the ceiling should be higher than 1.15 FXD and then positionCeiling needs to be adjusted after that?

Reply:
2023 April 22nd 8:22 PM
Anton : 50k will be _positionDebtVaue